capella2polarion/converters/converter_config.py

```diff
@@ -143,13 +180,17 @@ def set_layer_config(
                 type_prefix,
             )
             self.polarion_types.add(p_type)
-            links = self._force_link_config(
+            links = LinkConfig.generate_links_configs(
                 type_config.get("links", []), role_prefix
             )
+            converters = self._force_dict(type_config.get("serializer"))
+            assert self.__global_config.converters is not None
+            assert closest_config.converters is not None
             self._layer_configs[layer][c_type].append(
                 CapellaTypeConfig(
                     p_type,
-                    type_config.get("serializer") or closest_config.converters,
+                    self.__global_config.converters
+                    | (converters or closest_config.converters),
```

Comment:
This changes the current behavior as all types will also use serializers of the global config. Let's double check that this is documented in the Release notes